recipe/meta.yaml

```
@@ -24,6 +24,7 @@ requirements:
     - python
     - pip
     - setuptools
+    - setuptools_scm
```

```suggestion
    - setuptools_scm >=7.0.1
```
Should we reflect the bounds?
https://github.com/yt-project/unyt/blob/09fb30062e927744f8305b673a21322a21b91034/pyproject.toml#L4